libs/common/irohad_version.hpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LIBS_COMMON_IROHAD_VERSION_HPP
+#define LIBS_COMMON_IROHAD_VERSION_HPP
+
+namespace iroha {
+
+  extern const char *kGitPrettyVersion;
```

Comment:
Why a separate file?

Reply:
When these definitions change, only this small file has to be recompiled.